pkgs/top-level/all-packages.nix

```
@@ -2298,6 +2298,8 @@ in
 
   patdiff = callPackage ../tools/misc/patdiff { };
 
+  patool = python3Packages.patool;
```

only python applications should be exported in all-packages
```suggestion
  patool = with python3Packages; toPythonApplication patool;
```